api/turing/cluster/labeller/labeller.go

```diff
@@ -3,18 +3,18 @@ package labeller
 import "fmt"
 
 const (
-	// orchestratorValue is the value of the orchestrator (which is Turing)
+	// OrchestratorValue is the value of the orchestrator (which is Turing)
 	orchestratorValue = "turing"
-	// environmentLabel refers to the environment the Kubernetes object is in
+	// EnvironmentLabel refers to the environment the Kubernetes object is in
```

Comment:
Isn't it expected that comment matches the variable name?

Reply:
oops, I made it public then back to private and forgot to change it. My editor nags me when it's public and not exported with a comment but doesn't do the converse action.